protostar-rust/tests/e2e/running.rs

```diff
@@ -0,0 +1,25 @@
+use crate::common::runner::{corelib_path, runner};
+use assert_fs::fixture::PathCopy;
+use indoc::indoc;
+
+#[test]
+fn running_tests() {
+    let temp = assert_fs::TempDir::new().unwrap();
+    temp.copy_from("example_package", &["**/*"]).unwrap();
+
+    let snapbox = runner();
+    let corelib = corelib_path();
+
+    snapbox
+        .current_dir(&temp)
+        .args(&["--corelib-path", corelib])
```

Comment:
We probably should test the scenario without it being specified, is that possible?

Also I would us to have module level integration tests running some cairo code (as we did for cheatcodes), though it doesn't have to be included here necessarily

Reply:
We can write a test without corelib-path specified but it will simply fail: Runner still tries to detect the corelib when constructing the db, but this detection doesn't work with e2e tests.